packages/rancher-monitoring/rancher-service-metrics/charts/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,56 @@
+{{- $chartLabel := .Values.chartLabel }}
+{{- if eq $chartLabel "" }}
+{{- $chartLabel = (index (lookup "apps/v1" "Deployment" "cattle-system" "rancher").metadata.labels "chart") }}
+{{- end }}
+{{- if ne $chartLabel "" }}
```

Comment:
should we error out if we can't find rancher on attempting to install?

Reply:
Don't think this is necessary, monitoring can be installed without this and if label isn't present the service monitor will just be skipped.